packages/@react-aria/landmark/stories/Landmark.stories.tsx

```diff
@@ -312,6 +314,37 @@ function ApplicationExample() {
   );
 }
 
+function ChangeLabelExample() {
```

Comment:
@reidbarber yep, this works for me, perhaps even add a second Navigation landmark in this story that has an aria-label of "Second label" so that we can confirm that the landmark gets properly updated and subsequently triggers the checkLabels bit.